pkg/build/build.go

```diff
@@ -25,6 +26,17 @@ func BuildContainerDisk(imgPath string, checksum string) (v1.Image, error) {
 		return nil, fmt.Errorf("error appending the image layer: %v", err)
 	}
 
+	cf, err := img.ConfigFile()
+	if err != nil {
+		return nil, fmt.Errorf("error getting the image config file: %v", err)
+	}
+
+	cf.Architecture = ImageArchitecture
```

Comment:
Is there a way to infer the value of the destination container image architecture from the input artifacts architecture?